src/nvim/buffer.c

```diff
@@ -2833,15 +2816,16 @@ build_stl_str_hl (
       Trunc
     }               type;
   }           item[STL_MAX_ITEM];
-  int minwid;
-  int maxwid;
-  int zeropad;
-  char_u base;
-  char_u opt;
+
+  typedef enum {
+     DECIMAL     = 10,
+     OCTAL       = 8,
+     HEXIDECIMAL = 16
+  } number_base;
```

Comment:
Do you have a better solution than the enum?

I am fine updating to match the style guide, but if you have an alternate idea I'd also be happy to change it.